### PR TITLE
Update Cohort-call-facilitator-Guide.md to include latest addition

### PR DESCRIPTION
**Pull request by malvikasharan**
Include Paz and my updates on the guide

### CODE REVIEW

File: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md [hunk 1]
```diff
@@ -1,51 +1,103 @@
...
-#### Before the call
-Reviewing the cohort call notes (on Etherpad) for :
+Our cohort calls vary in time from week to week: Please see schedules from our website here: [[https://openlifesci.org/](https://openlifesci.org/)]([https://openlifesci.org/](https://openlifesci.org/)).
 
- - Typos and consistency (e.g. are all the headings formatted headings?)
- - Incorrect URLs
-    - In many cases we copy the notes document from the last cohort, so always good to watch out for links to previous cohorts’ slides/resources. We don’t expect you to have the correct slide links though - please just let the team know on Slack about these in the facilitator's channel 
- - Information that should be shared with the cohort prior to the scheduled cohort call
-**Time commitment:** 10-20 minutes per call
+For any questions or concerns, please contact Community Coordinator, Paz Berlando via Slack.
```
Comment (yochannah) [inline cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md:15]:
```suggestion
For any questions or concerns, please contact Community Coordinator, Paz Bernaldo via Slack.
```

File: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md [hunk 2]
```diff
@@ -1,51 +1,103 @@
...
- - Follow the list of tasks on the cohort call processing issue, under “4. cohort call video preparation”. An example: https://github.com/open-life-science/open-life-science.github.io/issues/283
- - Ask the organising team for credentials to [otter.ai](http://otter.ai/), YouTube and Zoom!
+1. **Call Facilitator** reviews the cohort call notes and attends the cohort call to help the cohort call lead in making sure that a cohort call in a smooth, kind, and timely manner
+2. **Recording Facilitator** reviews the recording from the cohort call by checking the otter.ai transcription for accuracy and sharing the video via Youtube
 
-**Time commitment:** ~2 hours per call, latest by 9 pm UK Time on the Monday after the call (so that we can send the weekly cohort emails with the link to the recording!)
+**Please ask for the signup sheet where you can indicate your availability for 1 or multiple of these roles**
 
-OLS team would like to offer an honorarium amount of $50 for each call you contribute to.
+As of August 2023, both facilitator roles are compensated with $ 50 USD per hour.
```
Comment (yochannah) [inline cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md:28]:
```suggestion
As of August 2023, both facilitator roles are compensated with $ 50 USD per hour, with a maximum of 1.5 hours per cohort call or 3 hours for transcription.
```

idk if we have this but maybe we should?

File: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md [hunk 3]
```diff
@@ -1,51 +1,103 @@
...
+2. Test if you can create breakout rooms (for discussion parts).
+3. Help manage participants by letting them in (if the waiting room is enabled) and mute them if their mic is on when they are not speaking to avoid background noise.
+4. Keep an eye on the chat and respond to questions. You can also privately coordinate with the participants or cohort call lead as needed, for example, if someone needs help with addressing any confusion or interruptions for attendees.
+5. Remind newcomers via the chat to add an_ S _or a _W_ in front of their Zoom name (S for spoken; W for written), for the breakout rooms. Privately coordinate with them if they have not indicated ‘W’ for written and ‘S’ for speaking to allow the creation of breakout rooms.
+6. Share the Etherpad link in the chat for the participants who arrive late and remind them to add their names in the roll call section of the Etherpad.
+7. Please also help take some notes per talk when you are not chairing.
```
Comment (yochannah) [inline cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md:62]:
```suggestion
7. Please also help take some notes about the talks when you are not chairing.
```

File: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md [hunk 4]
```diff
@@ -1,51 +1,103 @@
...
+8. During the call, share the Etherpad’s line number where the participants should respond to a written reflection, questions or comments for speakers, prompts for the breakout discussion or notes section from breakout sessions.
+9. Create the breakout rooms if needed, considering people’s choice for spoken or written rooms. 
+10. Chair the session that you have agreed to (you would have coordinated this in advance with the call leads or community coordinator)
```
Comment (yochannah) [inline cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md:65]:
```suggestion
10. Chair the session that you have agreed to (you would have coordinated this in advance with the call leads or community coordinator)

#### Chairing notes: 
- If you are chairing a speaker session, make sure to pay attention to time, and ask the speaker to wrap up if they look like they're going to over-run their talk time. 
- Always think of questions for the speakers. Often no-one wants to be the first person to ask, so a good call host can "break the ice" by asking one themselves. 
```

File: cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md [hunk 5]
```diff
@@ -1,51 +1,103 @@
...
+
+### Recording Facilitator
+
+**These facilitators hold the post-call responsibility mainly.**
+
+What you would need:
```
Comment (yochannah) [inline cohort-procedures-and-templates/Cohort-call-facilitator-Guide.md:71]:
```suggestion
What you will need: 
```

**Review by yochannah — APPROVED** [review]

**Review by iramosp — APPROVED** [review]

**Comment by iramosp** [comment]
Thanks everyone, this guide will also be useful for the Nebula cohorts. And sorry on the delayed review, I'll merge now.

**Pull request merged**